Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -207,11 +238,15 @@ void GeneralWidget::AddDescriptions()
       QT_TR_NOOP("Uses the main Dolphin window for rendering rather than "
                  "a separate render window.<br><br><dolphin_emphasis>If unsure, leave "
                  "this unchecked.</dolphin_emphasis>");
-  static const char TR_ASPECT_RATIO_DESCRIPTION[] = QT_TR_NOOP(
-      "Selects which aspect ratio to use when rendering.<br><br>Auto: Uses the native aspect "
-      "ratio<br>Force 16:9: Mimics an analog TV with a widescreen aspect ratio.<br>Force 4:3: "
-      "Mimics a standard 4:3 analog TV.<br>Stretch to Window: Stretches the picture to the "
-      "window size.<br><br><dolphin_emphasis>If unsure, select Auto.</dolphin_emphasis>");
+  static const char TR_ASPECT_RATIO_DESCRIPTION[] =
+      QT_TR_NOOP("Selects which aspect ratio to use when rendering.<br>"
+                 "Each game can have a slightly different native aspect ratio."
+                 "<br><br>Auto: Uses the native aspect ratio"
+                 "<br>Force 16:9: Mimics an analog TV with a widescreen aspect ratio."
+                 "<br>Force 4:3: Mimics a standard 4:3 analog TV."
```

Comment:
I know this is an old bit here, but while we are at it... the use of "Analog TV" here bugs me. Like, I get why that's there, it's assuming a "dumb" display that defaults to stretching, but that's not necessarily true even in analog.  Even the use of "standard" is silly. I'd much prefer:

>Force 4:3: Mimics a 4:3 display.
>Force 16:9 Mimics a display with a 16:9 widescreen aspect ratio.

That's all this needs really. 

Like, that's all this needs.

Reply:
I know what you want to say but I don't think "Ignores the game's configured aspect ratio" is quite right here either. Both of these options *are* taking the game's video configuration into account, they just ignore our heuristic to decide whether the game is trying to render 4:3 or 16:9.

Reply:
(this stuff is really hard to put into a short single-line string for users who don't have technical knowledge so I suppose it's okay to be wrong in order the get the point across, but yeah...)

Reply:
Doesn't Force 4:3 and Force 16:9 outright stretch to exactly 4:3 and 16:9?

Reply:
Yeah they don't, they behave exactly like 'Auto' except it skips the heuristic and just assumes its result.

Reply:
Yes, as much as I don't like the old wording, it kind of makes sense. If a game was meant for 4:3 output but played on a 16:9 CRT TV in 16:9 mode, the same distortion would now apply in Dolphin when you force 16:9. The opposite for 4:3 being forced on 16:9 games. Overall, these options are barely ever useful if not for people that use CTRs? Or for games that fail the heuristic aspect ratio detection I guess.
One thing that is confusing about their wording is that it's not exactly 16:9, at least not most of the times. So maybe they could be called `Force ~16:9` or something like that :D.
Though we can also fix these later without it blocking this PR, given I didn't really change these lines myself.

Reply:
Yeah... I'm thinking that this will need tweaking but at the same time I don't think it makes sense to hold up this PR over this. Go fix the other string and then after this is merged I'll open up a new PR for further discussion on the phrasing of this one, if that's fine with everyone.